bot/cogs/alias.py

```diff
@@ -34,19 +34,32 @@ def __init__(self, bot: Bot):
 
         await ctx.invoke(cmd, *args, **kwargs)
 
+    async def test_alias_can_run(self, ctx: Context, alias: str) -> bool:
```

Comment:
There probably are some `test_` prefixes scattered around in non-test code. But I personally use the `test_` prefix for only test cases. 
My suggestion is that we change the function name to something that does not use the `test_` prefix.